bxin/ptg2m2/m2_diagnostic_EFD.ipynb

```diff
@@ -239,7 +245,7 @@
    "outputs": [],
    "source": [
     "M2Weight = 15578 #Newton\n",
-    "def lookUpGravity(lutInAngle, FE, FA,elevA):\n",
+    "def lookUpGravity(summit, lutInAngle, FE, FA,elevA):\n",
```

Comment:
The doc string of `summit` might be needed. There are the similar problems for other functions (`def`).

Reply:
thanks. Fixed.